egs/babel_multilang/s5/run-4-anydecode-langs.sh

```diff
@@ -0,0 +1,471 @@
+#!/bin/bash
```

Comment:
I am not familiar with all the possible use-cases for this script. @jtrmal might be better suited to review this script.